name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/ABNAMROGroupPDFExtractor.java

```diff
@@ -36,15 +37,15 @@ private void addAccountStatementTransaction()
                                                         // @formatter:on
                                                         section -> section //
                                                                         .attributes("currency") //
-                                                                        .match("^Tagesgeldkonto \\(alle Betr.* in (?<currency>[\\w]{3})\\).*$") //
+                                                                        .match("^Tagesgeldkonto \\(alle Betr.* in (?<currency>[A-Z]{3})\\).*$") //
                                                                         .assign((ctx, v) -> ctx.put("currency", asCurrencyCode(v.get("currency")))),
                                                         // @formatter:off
                                                         // Tagesgeldkonto 100,00 24.10.2011 100,00
                                                         // @formatter:on
                                                         section -> section //
                                                                         .attributes("currency") //
                                                                         .match("^Tagesgeldkonto (?<currency>.*)$") //
-                                                                        .assign((ctx, v) -> ctx.put("currency", asCurrencyCode("EUR"))))
+                                                                        .assign((ctx, v) -> ctx.put("currency", asCurrencyCode(CurrencyUnit.EUR))))
```

Comment:
Instead of ```asCurrencyCode(CurrencyUnit.EUR))``` we can directly say ```CurrencyUnit.EUR```.
If we have a valid currency code (it is a static variable), then why try to resolve it it itself?